herokux/data_source_herokux_addons.go

```diff
@@ -0,0 +1,72 @@
+package herokux
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceHerokuxAddons() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceHerokuxAddonsRead,
+		Schema: map[string]*schema.Schema{
+			"app_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"addon_service_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"addon_ids": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+
+			"addon_names": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
```

Comment:
See my comment I left in the `docs/data-sources/addons.md` file regarding `addon_ids` and `addon_names`

Reply:
Updated